Dockerfile

```diff
@@ -49,6 +49,7 @@ RUN mkdir /app/log
 
 COPY --from=frontend-build /app/src/openzaak/static/css /app/src/openzaak/static/css
 COPY ./src /app/src
+COPY ./fixtures /app/fixtures
```

Comment:
Can we mount those as a volume instead of adding them to the docker image itself?

Reply:
@sergei-maertens I prefer to focus on getting the newman tests to pass.

@stevenbal Can keep it like this for now.

I created a new issue for this: https://github.com/open-zaak/open-zaak/issues/110